SPECS/openssl/openssl.spec

```diff
@@ -4,7 +4,7 @@
 Summary:        Utilities from the general purpose cryptography library with TLS implementation
 Name:           openssl
 Version:        1.1.1k
-Release:        30%{?dist}
+Release:        31%{?dist}
```

Comment:
Could you add more details to the PR's summary or inside the patch itself explaining why that fixes the CVE? [OpenSSL's GitHub](https://github.com/openssl/openssl/pull/13817) issue related to it seems to have been rejected. You mentioned [Ubuntu applied the change](https://bugs.launchpad.net/ubuntu/+source/openssl/+bug/2054090) but getting all of that figured out in the future, if needed, may take a while, so a comment is greatly appreciated.